### PR TITLE
osbuilder/qat: use centos as base OS

### PR DESCRIPTION
**Pull request by devimc**
move away from ubuntu, since now it's easier to build using
CentOS as base OS

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Review by GabyCT — APPROVED** [review]

**Review by fidencio — APPROVED** [review]
lgtm, thanks @devimc!

**Comment by fidencio** [comment]
"""
[osbuilder/qat: use centos as base OS]

move away from ubuntu since now it is more complex to build
using fedora container
"""

I'd rephrase it as:
"move away from ubuntu, since now it's easier to build using CentOS as base OS".

**Comment by fidencio** [comment]
Adding the `do-not-merge` label so @devimc can reword the PR.

**Comment by devimc** [comment]
change applied, thanks @fidencio 
/test
